doc/source/whatsnew/v1.2.0.rst

```diff
@@ -0,0 +1,210 @@
+.. _whatsnew_120:
```

Comment:
Same with this one (since you opened up the other PR with the new whatsnew file)